pkg/serviceaccount/jwt_test.go

```diff
@@ -304,22 +303,3 @@ func TestTokenGenerateAndValidate(t *testing.T) {
 		}
 	}
 }
-
-func TestMakeSplitUsername(t *testing.T) {
-	username := apiserverserviceaccount.MakeUsername("ns", "name")
-	ns, name, err := apiserverserviceaccount.SplitUsername(username)
-	if err != nil {
-		t.Errorf("Unexpected error %v", err)
-	}
-	if ns != "ns" || name != "name" {
-		t.Errorf("Expected ns/name, got %s/%s", ns, name)
-	}
-
-	invalid := []string{"test", "system:serviceaccount", "system:serviceaccount:", "system:serviceaccount:ns", "system:serviceaccount:ns:name:extra"}
-	for _, n := range invalid {
-		_, _, err := apiserverserviceaccount.SplitUsername("test")
```

Comment:
this test is invalid.
apiserverserviceaccount.SplitUsername("test") -> apiserverserviceaccount.SplitUsername(n)

It should be removed anyway.